### PR TITLE
data/alicloud_cs_kubernetes_clusters, alicloud_cs_managed_kubernetes_clusters, alicloud_cs_serverless_kubernetes_clusters: support exporting kube config file

### PR DESCRIPTION
**Pull request by Pangjiping**
rewrite common function which gets kube config information for ACK

### CODE REVIEW

File: alicloud/data_source_alicloud_cs_kubernetes_clusters.go [hunk 1]
```diff
@@ -670,3 +693,17 @@ func csKubernetesClusterDescriptionAttributes(d *schema.ResourceData, clusterTyp
 
 	return nil
 }
+
+// getKubeConfig get cluster kube config info.
```
Comment (Wanghx0991) [inline alicloud/data_source_alicloud_cs_kubernetes_clusters.go:697]: 抽出的部分, 放到`service_alicloud_cs.go` 里

File: website/docs/d/cs_kubernetes_clusters.html.markdown [hunk 2]
```diff
@@ -70,6 +70,7 @@ The following attributes are exported in addition to the arguments listed above:
     * `type` - Type of collecting logs.
     * `project` - Log Service project name.
   * `resource_group_id` - (Optional, ForceNew, Available in 1.101.0+) The ID of the resource group,by default these cloud resources are automatically assigned to the default resource group.
+  * `kube_config` - (Optional, Available in 1.173.0+) Kubernetes cluster kube config file prefix. For given `kube_config`, the kube config credentials will be stored with the following file name `kube_config-clusterid-kubeconfig`.
```
Comment (Wanghx0991) [inline website/docs/d/cs_kubernetes_clusters.html.markdown:73]: 标注下不指定`kube_config` 文件名样式

Comment (Pangjiping) [reply website/docs/d/cs_kubernetes_clusters.html.markdown]: done

Comment (Pangjiping) [reply website/docs/d/cs_kubernetes_clusters.html.markdown]: done

File: alicloud/data_source_alicloud_cs_kubernetes_clusters.go (hunk 1 above)
Comment (Pangjiping) [reply alicloud/data_source_alicloud_cs_kubernetes_clusters.go]: done

File: alicloud/data_source_alicloud_cs_kubernetes_clusters.go [hunk 3]
```diff
@@ -651,6 +657,19 @@ func csKubernetesClusterDescriptionAttributes(d *schema.ResourceData, clusterTyp
 			mapping["nat_gateway_id"] = nat.NatGateways.NatGateway[0].NatGatewayId
 		}
 
+		// kube_config
+		var kubeConfig *cs.ClusterConfig
+		filePath := fmt.Sprintf("%s-kubeconfig", ct.ClusterID)
+		if kubeConfig, err = csService.DescribeClusterKubeConfig(ct.ClusterID, false); err != nil {
+			return WrapError(err)
+		}
+		if filePrefix, ok := d.GetOk("kube_config"); ok && filePrefix.(string) != "" {
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_cs_kubernetes_clusters.go:666]: 如果是前缀的话，应该是 kube_config_file_prefix

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**